src/Workspaces/Core/Portable/project.json

```diff
@@ -1,8 +1,24 @@
 ﻿{
-  "dependencies": {
-    "Newtonsoft.Json": "6.0.4"
-  },
-  "frameworks": {
-    ".NETPortable,Version=v4.5,Profile=Profile7": {}
-  }
+    "supports": {
+        "net46.app": {}
+    },
+    "dependencies": {
+        "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+        "Newtonsoft.Json": "6.0.4",
+        "System.Threading.Tasks": "4.0.11",
```

Comment:
Yes.  Forget the exact version to use though. @tmat Any objection to doing that here?